File: bronze/1159/c++/main.cpp
#include <iostream>
#include <string>
#include <map>

int main()
{   
    int num;
    std::cin >> num;
    std::map<char, int> list;
    std::string in;
    std::string answer = "";
    char temp;
    for (int i = 0; i < num; i++) {
        std::cin >> in;
        temp = in[0];
        if (list.find(temp) != list.end()) {
            list[temp] += 1;
        } else {
            list[temp] = 1;
        }
    }
    for (const auto& data : list) {
        if (data.second >= 5) {
            answer += data.first;
        }
    }

    std::cout << ((answer.length() > 0) ? answer : "PREDAJA");
    
    return 0;
}

File: bronze/1159/python/main.py

if (__name__ == "__main__"):
    num = int(input())
    result = dict()
    for i in range(num):
        temp = input()[0]
        if (result.get(temp) is not None):
            result[temp] += 1
        else :
            result[temp] = 1
    sorted_key = dict(sorted(result.items(), key=lambda x: x[0]))
    answer = str()
    for data in sorted_key:
        if (result[data] >= 5):
            answer += data
    print(answer if len(answer) else "PREDAJA")
